pubspec.lock

```diff
@@ -28,7 +28,7 @@ packages:
       name: async
```

Comment:
mmh, does this pull request require updating the `pubspec.lock`? If not, it would be best to remove these updates.